teuthology/provision.py

```diff
@@ -29,73 +30,147 @@ def _get_downburst_exec():
     import pwd
     little_old_me = pwd.getpwuid(os.getuid()).pw_name
     for user in [little_old_me, 'ubuntu', 'teuthology']:
-        pth = os.path.expanduser("~%s/src/downburst/virtualenv/bin/downburst"
-                                 % user)
+        pth = os.path.expanduser(
+            "~%s/src/downburst/virtualenv/bin/downburst" % user)
```

Comment:
I think it should mostly just look in $PATH and not feel responsible for poking around in other locations, but I also think the scope of this PR is large enough ;)